### ISSUE TITLE
Ideas on how to design the crop showcase tab

### ISSUE THREAD

**Issue by jwlebert** [jwlebert/mfgjs2023#9]
I'm thinking it would be best if we limited use of text and instead made use of little pictograms. We would provide a tutorial explaining it which would be basically a better drawn version of the attached diagram. We can demonstrate what seasons the crop can grow in with the little pictograms.

![possible shop showcase](https://github.com/jwlebert/mfgjs2023/assets/37757770/d20b8033-7ec9-4f10-96b1-c44c586dc4e4)

**Comment by cmdbris** [comment]
Good idea, maybe consider putting the water interval and growth period underneath the value and cost, rather than side-by-side to avoid confusion
